docker/python3/Dockerfile

```
@@ -1,5 +1,5 @@
-# Last modified: Thu, 28 May 2020 22:53:09 +0000
-FROM python:3.8.3-alpine3.11
+# Last modified: Wed, 22 Jul 2020 08:27:22 +0000
+FROM python:3.8.5-alpine3.11
```

Let's also bump the alpine version to 3.12
```suggestion
FROM python:3.8.5-alpine3.12
```